include/inviwo/core/datastructures/buffer/buffer.h

```diff
@@ -38,9 +38,12 @@
 
 namespace inviwo {
 
+class DataFormatBase;
+
 class IVW_CORE_API BufferBase : public Data<BufferBase, BufferRepresentation> {
 public:
-    BufferBase(size_t size, const DataFormatBase* format, BufferUsage usage, BufferTarget target);
+    BufferBase(size_t defaultSize, const DataFormatBase* defaultFormat, BufferUsage usage,
```

Comment:
There probably needs to be some explanation of what default size and format means

Reply:
it should be pretty obvious from the naming. The only time these parameters are used for is the creation of a default representation.

Reply:
if you want, feel free to add comments. But in a separate PR.